automation_tools/satellite6/log.py

```diff
@@ -115,8 +133,12 @@ def analyze(log_file, content):
 
     def print_lines(lines_enumeration):
         """print lines with numbers"""
+        no_line_flag = True
         for i, line in lines_enumeration:
             _print_wrapper('{}: {}'.format(i, line))
+            no_line_flag = False
+        if no_line_flag:
+            _print_wrapper('None')
```

Comment:
I this is not required to print 'None'. We can leave as it is or we can return 'False' and operate accordingly.